### DISCUSSION TITLE
rename command option in 3.6 Jedis

### DISCUSSION THREAD

**Discussion by gsuda2001** [xetorthio/jedis#2570] [Q&A] [answered]
Hi ,

We need to rename some command in Redis due to security reasons. Please let us know how to configure renamed command in Jedis clients. Does Jedis client support rename commands options?

**Comment by sazzad16** [comment, marked as answer]
Jedis does not support command renaming.

FYI, Redis has deprecated command renaming.

```
# Command renaming (DEPRECATED).
#
# ------------------------------------------------------------------------
# WARNING: avoid using this option if possible. Instead use ACLs to remove
# commands from the default user, and put them only in some admin user you
# create for administrative purposes.
# ------------------------------------------------------------------------
```
Source: https://github.com/redis/redis/blob/unstable/redis.conf

It is suggested to use ACL to control the access of commands. Jedis supports ACL.

**Reply by gsuda2001** [reply]
We have upgraded to 6.x Redis for our application and started using ACL options and deprecated command renaming options which we followed till 5.x Redis.
